codespell_lib/data/dictionary.txt

```diff
@@ -44657,6 +44670,10 @@ redandant->redundant
 redction->reduction, redaction,
 redeable->readable
 redeclaation->redeclaration
+redeclare->re-declare
```

Comment:
* [OED](https://www.oed.com/dictionary/redeclare_v) shows `redeclare` is a valid word.
* [The Blue Book of Grammar and Punctuation](https://www.grammarbook.com/blog/hyphens/hyphens-with-the-prefix-re/), among other sources, recommends to **avoid the hyphen** in most cases.
* The[ Google Ngram Viewer](https://books.google.com/ngrams/graph?content=redeclare%2Cre-declare&year_start=1800&year_end=2019&corpus=en-2019) shows that `redeclare` is actually more common.

Reply:
```suggestion
```

Reply:
I did implement the opposite following the message above.
Now re-declare maps to redeclare (same goes for all verbal forms).